rollup.config.js

```diff
@@ -1,14 +1,63 @@
 import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from 'rollup-plugin-typescript';
+import alias from '@rollup/plugin-alias';
+import path from 'path';
 
-export default {
-    input: 'src/index.ts',
-    plugins: [resolve(), commonjs(), typescript()],
-    external: ['prettier', 'svelte'],
-    output: {
-        file: 'plugin.js',
-        format: 'cjs',
-        sourcemap: true,
+const srcDir = path.resolve(__dirname, 'src');
```

Comment:
do we want to use svelte naming convetions?
```suggestion
const src_dir = path.resolve(__dirname, 'src');
```